### PR TITLE
Update misc tools

### PR DESCRIPTION
**Pull request by jacob-delgado**
DMN for now until more tools need to be updated.

### CODE REVIEW

File: docker/build-tools/Dockerfile [hunk 1]
```diff
@@ -49,14 +49,14 @@ ENV GOLANG_GRPC_PROTOBUF_VERSION=v1.2.0
 ENV GOLANGCI_LINT_VERSION=v1.51.2
 ENV HADOLINT_VERSION=v2.12.0
 ENV HELM3_VERSION=v3.11.1
-ENV HUGO_VERSION=0.110.0
+ENV HUGO_VERSION=0.111.0
```
Comment (ericvn) [inline docker/build-tools/Dockerfile:52]: I think 0.111.2 is the most recent. I know there is a PR to move up to that in istio.io.

**Comment by ericvn** [comment]
Also I see trivy went to 0.38.2

**Review by howardjohn — APPROVED** [review]

**Review by ericvn — DISMISSED** [review]

**Pull request merged**